clients/finance/index.ts

```diff
@@ -0,0 +1,93 @@
+import { HttpNotFound } from '#clients/exceptions';
```

Comment:
We should actually move to `client/ods` folder

About the name : how about `client/ods/donnees-financieres.ts`